docs/documentation/api-specification/mi-api.md

```diff
@@ -27,7 +27,136 @@ Checklist for requesting approval for your extensions.
 
 ---
 ## Description
+MI API allows to write and overwrite parameters sent into transactions. Supports both single and multiple transactions, custom logic and validation.  Can be used to abort transaction based on own validation. It is not possible to add extra fields to transactions. Provides error managment. 
 
 ## Features
 
+### Getting value from data container
```

Comment:
For this documentation we should add more examples on how to treat different input data types int, String, etc.